src/index.html

```diff
@@ -15,17 +15,17 @@ <h1>Leaderboard</h1>
         <div class="mx-2">
             <div class="d-flex justify-content-center">
                 <h3 class="mx-3">Recent scores</h3>
-                <button id="refeshData">Refresh</button>
+                <button id="refeshData" class="btn btn-warning">Refresh</button>
             </div>
             <div id="scoresTable" class="my-3 border border-2 border-dark">
     
             </div>
         </div>
         <div class="d-flex flex-column mx-2">
             <h3>Add your score</h3>
-            <input id="userName" class="my-2" type="text" placeholder="Your name">
-            <input id="userScore" class="my-2" type="text" placeholder="Your score">
-            <button id="scoresSubmit" class="my-2 w-50 ms-auto">Submit</button>
+            <input id="userName" class="my-2 border border-warning" type="text" placeholder="Your name">
+            <input id="userScore" class="my-2 border border-warning" type="text" placeholder="Your score">
+            <button id="scoresSubmit" class="my-2 w-50 ms-auto btn btn-warning">Submit</button>
         </div>
     </div>
 </body>
```

Comment:
- [ ] (optional) The general layout of the app is good. It would even improve your design more if you could reduce the amount of margin between the scores section and the start of the page in horizontal view so as to match the given wireframe more.

Your design
![image](https://github.com/alexansaa/LeaderBoard/assets/81932478/4fb6ef83-c70a-436a-9117-4199806c61fb)

Given design
![image](https://github.com/alexansaa/LeaderBoard/assets/81932478/8e9fcb19-8113-4933-b901-ea2323c7adf1)